projectDocs/translating/crowdin.md

```diff
@@ -0,0 +1,100 @@
+
+1. Download the latest Windows PoEdit version at <https://poedit.net/download>
+1. Install it by following the on-screen instructions, the default options should be sufficient.
+1. When launching PoEdit:
+    1. Choose "Translate cloud project"
+    1. Connect your Crowdin account
+    1. Select NVDA and the language you wish to translate
+
+## Translating using PoEdit
+
+After opening a .po file you will be placed on a list with all of the strings to translate.
+
+You can read the status bar to see how many strings have already been translated, the number of untranslated messages, and how many are fuzzy.
+A fuzzy string is a message which has been automatically translated, thus it may be wrong.
+PoEdit will collect the new and fuzzy messages and presents them at the top of the already translated messages.
+
+To insert or correct the translation for a string, first select it with the arrows, then tab to the blank edit field and type its translation.
+
+NVDA will beep if you are on an untranslated or fuzzy message.
+If you are using a braille display you'll see a star sign in-front of the messages you have to translate.
+
+You may want to spell the original string to be aware of any punctuation mark, capital letters, etc.
+PoEdit has a keystroke you may press while on an original string, `alt+c`, that copies the original string to the edit field when pressed.
+You may then replace it with your translation as normal.
+
+Press `control+s` at any moment to save your work.
+Each time you press this key, PoEdit saves the po file, and if you check compile mo file checkbox in preferences, the .mo file will be re-generated.
+
+NVDA provides additional shortcuts for PoEdit which are described in [the User Guide](https://www.nvaccess.org/files/nvda/documentation/userGuide.html#Poedit).
+
+If you are unsure of meaning of the original interface message, consult automatic comments (also called translator comments).
+Some comments provide an example output message to help you understand what NVDA will say when speaking or brailling such messages.
+
+## Translating the interface
+
+Open "nvda.po" for the language you want to translate in PoEdit.
+Alternatively, you can use Crowdin's interface directly.
+
+### Messages with formatting strings
+
+You will come across several messages that have additional characters or punctuation as part of the message, this section will explain how they should be treated.
+
+- `%d`, digits
+- `%s`, string
+- `%.2f`, a number to 2 digits after the comma, for example 5.25
+- `{keyword}`, the text around the keyword should be translated, but the word should be left as is.
+
+#### Examples
+
+- `%d percent`: this means that `%d` will be replaced by a number when the program is running, and you only need to translate the word `percent`.
+- `subject: %s`: here `%s` means that another string will be substituted.
+- `on {backgroundColor}`: In this case, the word on should be translated, and the rest should be left alone.

```

Comment:
With the real string in `nvda.po`, matching the example:
```suggestion
- `{color} on {backgroundColor}`: In this case, the word on should be translated, and the rest should be left alone.
```

or even with the word "on" formatted:
```suggestion
- `{color} on {backgroundColor}`: In this case, the word `on` should be translated, and the rest should be left alone.
```

Reply:
```suggestion
- `{color} on {backgroundColor}`: In this case, the word `on` should be translated.
The rest should be left alone or rearranged in order to suit the language.
```